cluster/ubuntu/util.sh

```diff
@@ -508,6 +505,12 @@ function provision-masterandnode() {
       cp ~/kube/minion/* /opt/bin/
 
       service etcd start
+      service kube-apiserver start
```

Comment:
I think this script is just a script for newbies. Newbies can not set up a cluster without these lines, according to the offical document. They, like me, are just in trouble. 

What does 'triggered by etcd' mean? Will `etcd` start them automatically?